### DISCUSSION TITLE
How to make my blog reachable with google search index and google analytics

### DISCUSSION THREAD

**Discussion by abdelkareemkobo** [quarto-dev/quarto-cli#5475] [Q&A] [answered]
I found after some time that my blog post are not found in google search ! 
How to connect my blog with Welcome to Google Search Console to verify this is my webiste !
for google analytics i read the documents and add the analytics tag into _quarto.yaml in the right place 
is there any thing should i do to make my pages indexed and found in the search!
I know that i am using just github pages :)

**Comment by mcanouil** [comment, marked as answer]
It's not something on Quarto's side.
It's basic internet fiddling, see for example <https://search.google.com/search-console/about>.

Also for new website, it can take some time before robots come crawling yours.
